activerecord/lib/active_record/store.rb

```diff
@@ -44,6 +44,17 @@ module ActiveRecord
   #     store_accessor :settings, :privileges, :servants
   #   end
   #
+  #   # Optionally specify prefix/suffix to accessor with _prefix/_suffix option
+  #   class User
+  #     store :settings, accessors: [ :two_factor_auth ], coder: JSON, _prefix: true
+  #     store_accessor :settings, :secret_question, _prefix: 'config'
+  #     store_accessor :settings, :login_retry, _suffix: 'setting'
```

Comment:
Should the examples be using symbols instead of strings for `_prefix` and `_suffix` values?